api/v1alpha1/bundle_types.go

```diff
@@ -81,6 +81,11 @@ type GitSource struct {
 	// is required. Setting more than one field or zero fields will result in an
 	// error.
 	Ref GitRef `json:"ref"`
+	// SecretName contains the secret name that has authorization information for HTTPS protocol and is in the namespace that the provisioner is deployed.
+	// The secret is expected to contain `data.username` and `data.accesstoken` for the username and personal access token, respectively.
+	SecretName string `json:"secretName,omitempty"`
+	// If TslNoVerify is set true, the server certificate is ignored.
```

Comment:
Nit: GoDoc should start with the Identifier: `// TLSNoVerify ...` There's also a typo in the spelling of `TslNoVerify` in the comment.

Secondly, another nit: What should we call this field?
- `wget` uses `--no-check-certificate`
- `curl` uses `--insecure`
- Go Stdlib uses `InsecureSkipVerify`

Anyone else have examples of how different things refer to this concept?

I'm partial to using `Insecure` as the prefix, at least. That makes it clearer that this is a generally frowned-upon field to use (and we should state that in our GoDoc comment).